tigerpath/majors_and_certificates/majors/EEB.json

```diff
@@ -48,10 +48,31 @@
               "max_counted": 1,
               "min_needed": 1,
               "explanation": null,
-              "course_list": [
-                "EEB 214/MOL 214",
-                "EEB 215/MOL 215"
+              "req_list": [
+                {
+                  "name": null,
+                  "max_counted": 1,
+                  "min_needed": 1,

```

Comment:
https://eeb.princeton.edu/undergraduate/plan-study-0

"ISC Curriculum: Students who have completed the first full year of ISC are credited for MOL 214, CHM 201/2, and PHY 101/2. The student must take EEB 211. Students who complete 2 years of ISC can submit a short essay after reading Serengeti Rules and Selfish Gene to opt out of EEB 211."